src/Sulu/Bundle/MarkupBundle/Markup/HtmlMarkupParser.php

```diff
@@ -60,12 +60,17 @@ public function validate($content, $locale)
         $result = [];
         $tagMatchGroups = $this->tagExtractor->extract($content);
         foreach ($tagMatchGroups as $tagMatchGroup) {
-            $tags = $this->tagRegistry->getTag($tagMatchGroup->getTagName(), 'html', $tagMatchGroup->getNamespace())
+            $tags = $this->tagRegistry->getTag($tagMatchGroup->getTagName(), $this->getType(), $tagMatchGroup->getNamespace())
                 ->validateAll($tagMatchGroup->getTags(), $locale);
 
             $result = array_merge($result, $tags);
         }
 
         return $result;
     }
+
+    public function getType(): string
```

Comment:
@alexander-schranz I guess we cannot take this method into the interface because of bc right?

Reply:
Would not add this method just index by the key of the tagged service. See https://symfony.com/blog/new-in-symfony-4-3-indexed-and-tagged-service-collections